specification/configuration.md

```diff
@@ -130,6 +136,32 @@ beyond the OpenTelemetry specification exist.
+
+| Configuration property (default value) | Description                                                                                                                                                                                                 | Required |
+| -------------------------------------- | -----------                                                                                                                                                                                                 | -------- |
+| `realm` ()                             | Splunk realm, e.g. `us0`, `us1`. If set, value of `beaconUrl` will be automatically computed based on this. [1] [2]                                                                                         | Yes      |
+| `beaconUrl` ()                         | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. If both `realm` and `beaconUrl` are set, `beaconUrl` takes precedence. [1] [2]                                                        | Yes      |
+| `rumAuth` ()                           | RUM authentication token. [1]                                                                                                                                                                               | Yes      |
+| `applicationName` ()                   | Instrumented application name. [1]                                                                                                                                                                          | Yes      |
+| `globalAttributes` ()                  | OpenTelemetry [Attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes) that will be added to every span produced by the RUM library. | Yes      |
+| `allowInsecureBeacon` (false)          | Allows sending data to insecure endpoints not using https. [2]                                                                                                                                              | No       |
+| `deploymentEnvironment` ()             | Sets the environment (`deployment.environment` span attribute) for all spans.                                                                                                                               | Yes      |
+
+- [1] Application name, authentication token and either realm or the beacon URL MUST be provided by the user.
+  If any of these is missing, the RUM instrumentation library MUST fail to start.
+- [2] Implementations MUST enforce by default that `beaconUrl`s are https only, and reject/fail to start otherwise.
+  Implementations MAY offer an `allowInsecureBeacon` option (default false) for customers wishing to use that.
+
+Other requirements:
+- RUM library MUST use the Zipkin span exporter by default
+- RUM library MUST limit the number of sent spans to 100 in a 30 second window per `component` attribute value
```

Comment:
Is this customizable (up or down)? If yes, what happens if you exceed?

Reply:
In the first go-around, it's not configurable, but it would be easy to make it so. Spans get dropped beyond the limit.